src/Microsoft.ML/CSharpApi.cs

```diff
@@ -3804,6 +4171,228 @@ public AveragedPerceptronBinaryClassifierPipelineStep(Output output)
         }
     }
 
+    namespace Trainers
+    {
+
+        /// <summary>
+        /// Train binary ensemble.
+        /// </summary>
+        public sealed partial class BinaryEnsemble : Microsoft.ML.Runtime.EntryPoints.CommonInputs.ITrainerInputWithLabel, Microsoft.ML.Runtime.EntryPoints.CommonInputs.ITrainerInput, Microsoft.ML.ILearningPipelineItem
```

Comment:
>BinaryEnsemble [](start = 36, length = 14)

Do we want to adhere to the naming convention that trainers that are binary classifiers end with `BinaryClassifier`, and so forth? I think you followed that convention in your OVA PR #363.